### DISCUSSION TITLE
Bug with one repository

### DISCUSSION THREAD

**Discussion by ruth3n** [community/community#38810] [Mobile] [answered]
![image](https://user-images.githubusercontent.com/108727970/201198760-e546cdb5-d6f5-44fc-b530-cc022afa36c4.png)
Can't open repository, got this error every time.
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.88.0 (82868297)
Device: iPhone 11

**Comment by grahamzemel** [comment, marked as answer]
Hey there, it looks like your repository either isn't public or hasn't been synced with Github's servers and there may have been an error. 

If you have access to the internet via cellular or wifi, this should have gone away in the couple of hours since you asked the question. 

Hope that helps solve the issue if it wasn't solved already!

**Reply by Evil0ctal** [reply]
I have a same issue here:

https://github.com/orgs/community/discussions/39496
